bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/package-info.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+@org.osgi.annotation.bundle.Header(name = org.osgi.framework.Constants.BUNDLE_NATIVECODE, value = "lib/armv6hf/libjavatinyb.so;lib/armv6hf/libtinyb.so;processor=arm;osname=linux")
```

Comment:
It looks like this makes it impossible to run on a x86-64 environment

Reply:
Is that from looking at the code or testing? Because this is the same native-code setting that was used before the migration.

Reply:
This is the error i'm getting, but I have an intel cpu:

`[core.karaf.internal.FeatureInstaller] - Failed installing 'openhab-binding-bluetooth': Error restarting bundles:
        Could not resolve module: org.openhab.binding.bluetooth.bluez [301]
  Unresolved requirement: Require-Capability: osgi.native; native.paths:List<String>="lib/armv6hf/libjavatinyb.so,lib/armv6hf/libtinyb.so"; filter:="(&(osgi.native.osname~=linux)(osgi.native.processor~=arm))"`

Reply:
I found a small error. Can you update the bluez-bundle?

1. get the bundle number: `bundle:list | grep -i bluez`, the number in the first column is the bundle number
2. update bundle: `bundle:update <number> https://janessa.me/esh/org.openhab.binding.bluetooth.bluez-2.5.0-SNAPSHOT.jar`
3. restart the bundle `bundle:restart <number>`

and report back if it works.

Reply:
yes, the bluez bundle becomes active now! very nice.

but i get another exception, probably not related to this PR anymore:
`2019-07-07 22:45:49.713 [ERROR] [nal.common.AbstractInvocationHandler] - An error occurred while calling method 'ThingHandler.initialize()' on 'org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler
@70a5a067': null
java.lang.NullPointerException: null
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:101) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor61.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [136:org.openhab.core:2.5.0.201907020301]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [136:org.openhab.core:2.5.0.201907020301]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
2019-07-07 22:45:49.772 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while initializing handler of thing 'bluetooth:bluez:c8f3ddcc': null
java.lang.NullPointerException: null
        at org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler.initialize(BlueZBridgeHandler.java:101) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor61.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.eclipse.smarthome.core.internal.common.AbstractInvocationHandler.invokeDirect(AbstractInvocationHandler.java:153) [136:org.openhab.core:2.5.0.201907020301]
        at org.eclipse.smarthome.core.internal.common.Invocation.call(Invocation.java:53) [136:org.openhab.core:2.5.0.201907020301]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:?]
        at java.lang.Thread.run(Thread.java:748) [?:?]
2019-07-07 22:45:49.789 [ERROR] [org.openhab.binding.bluetooth.bluez ] - bundle org.openhab.binding.bluetooth.bluez:2.5.0.201907061738 (286)[org.openhab.binding.bluetooth.bluez.internal.discovery.BlueZDisco
veryService(336)] : The activate method has thrown an exception
java.lang.NullPointerException: null
        at org.openhab.binding.bluetooth.bluez.internal.discovery.BlueZDiscoveryService.startScan(BlueZDiscoveryService.java:64) ~[?:?]
        at org.openhab.binding.bluetooth.bluez.internal.discovery.BlueZDiscoveryService.startBackgroundDiscovery(BlueZDiscoveryService.java:51) ~[?:?]
        at org.eclipse.smarthome.config.discovery.AbstractDiscoveryService.activate(AbstractDiscoveryService.java:386) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:?]
        at org.apache.felix.scr.impl.inject.methods.BaseMethod.invokeMethod(BaseMethod.java:228) ~[42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.inject.methods.BaseMethod.access$500(BaseMethod.java:41) ~[42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.inject.methods.BaseMethod$Resolved.invoke(BaseMethod.java:664) ~[42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.inject.methods.BaseMethod.invoke(BaseMethod.java:510) [42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.inject.methods.ActivateMethod.invoke(ActivateMethod.java:317) [42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.inject.methods.ActivateMethod.invoke(ActivateMethod.java:307) [42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.createImplementationObject(SingleComponentManager.java:340) [42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.createComponent(SingleComponentManager.java:114) [42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:982) [42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getServiceInternal(SingleComponentManager.java:955) [42:org.apache.felix.scr:2.1.16]
        at org.apache.felix.scr.impl.manager.SingleComponentManager.getService(SingleComponentManager.java:900) [42:org.apache.felix.scr:2.1.16]
        at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse$1.run(ServiceFactoryUse.java:212) [?:?]`